vignettes/pull_request_comments.Rmd

```diff
@@ -0,0 +1,249 @@
+---
+title: "Download GitHub Pull Request Comments via REST API"
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteIndexEntry{Download GitHub Pull Request Comments via REST API}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+# Introduction
+
+In this vignette, the goal is to understand and visualize comments on
+the GitHub Pull Requests Conversations tab. The goal is to see the
+development process that involves communication between developers. The
+three REST API endpoints being used are issue comments, pull request
+comments, and pull request reviews.
+
+## Create a Personal Token
+
+GitHub limits the number of API calls per IP to only 60 attempts **every
+hour** at the time this vignette was created. You can check the current
+rates at [its
+website](https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#rate-limiting).
+
+If using a personal account token from a free GitHub account, the number
+of API calls per hour increases to 5000 **per hour**. Therefore, it is
+recommended you create a personal token by following the [GitHub
+Documentation
+instructions](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token#:~:text=Creating%20a%20token.%201%20Verify%20your%20email%20address%2C,able%20to%20see%20the%20token%20again.%20More%20items).
+The process should not take more than 2 minutes.
+
+The token should then be copied and pasted into `../.ssh/github_token`.
+Both the `.ssh` folder and `github_token` file may need to be created
+manually. Check for hidden folders before creating the `.ssh` folder.
+The `github_token` file should not have any file extensions.
```

Comment:
Instead of having this section here, move it closer to the R code on its own block that reads it. A second comment below pointing where. (*)